ccd-definition/CaseEvent/CLOSED.json

```diff
@@ -23,30 +23,13 @@
     "ID": "uploadDocuments-CLOSED",
     "Name": "Documents",
     "Description": "Upload documents",
-    "DisplayOrder": 9,
+    "DisplayOrder": 12,
     "PreConditionState(s)": "CLOSED",
     "PostConditionState": "CLOSED",
     "SecurityClassification": "Public",
     "ShowSummary": "N",
     "ShowEventNotes": "Y",
     "EndButtonLabel": "Save and continue",
     "Comment": "Recreated due to differing permissions in this state compared to other states"
-  },
-  {
-    "LiveFrom": "01/01/2017",
-    "CaseTypeID": "CARE_SUPERVISION_EPO",
-    "ID": "otherAllocationDecision-CLOSED",
```

Comment:
Although I highly doubt it with the funcitonality only just being released, you should probably make sure that the event hasn't been used in prod yet

Reply:
As discussed we have moved this to the DEPRECATED file and I will run this query separately and remove if not used